COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
The fact that Eelco is the only person mentioned in that file is a historical curiosity. He started the project, he created that file, and he's the only one who has ever touched it since (in 6b0b4586945e3531a04561f4fcffd7e5f666513d). IMHO, the contents of that file is inaccurate and meaningless, at best, and we need to improve the way we handle copyright issues.

Anyway, I agree with @jagajaga that adding Google to that license statement seems odd and I'm not in favor of doing so.